CefSharp.Core/ResourceHandlerWrapper.cpp

```diff
@@ -64,49 +65,6 @@ namespace CefSharp
 
     Cookie^ ResourceHandlerWrapper::GetCookie(const CefCookie& cefCookie)
     {
-        auto cookie = gcnew Cookie();
-        String^ cookieName = StringUtils::ToClr(cefCookie.name);
-
-        if (!String::IsNullOrEmpty(cookieName))
-        {
-            cookie->Name = StringUtils::ToClr(cefCookie.name);
-            cookie->Value = StringUtils::ToClr(cefCookie.value);
-            cookie->Domain = StringUtils::ToClr(cefCookie.domain);
-            cookie->Path = StringUtils::ToClr(cefCookie.path);
-            cookie->Secure = cefCookie.secure == 1;
-            cookie->HttpOnly = cefCookie.httponly == 1;
-
-            if (cefCookie.has_expires)
-            {
-                auto expires = cefCookie.expires;
-                cookie->Expires = DateTimeUtils::FromCefTime(expires.year,
-                    expires.month,
-                    expires.day_of_month,
-                    expires.hour,
-                    expires.minute,
-                    expires.second,
-                    expires.millisecond);
-            }
-
-            auto creation = cefCookie.creation;
-            cookie->Creation = DateTimeUtils::FromCefTime(creation.year,
-                creation.month,
-                creation.day_of_month,
-                creation.hour,
-                creation.minute,
-                creation.second,
-                creation.millisecond);
-
-            auto lastAccess = cefCookie.last_access;
-            cookie->LastAccess = DateTimeUtils::FromCefTime(lastAccess.year,
-                lastAccess.month,
-                lastAccess.day_of_month,
-                lastAccess.hour,
-                lastAccess.minute,
-                lastAccess.second,
-                lastAccess.millisecond);
-        }
-
-        return cookie;
+        return TypeConversion::FromNative(cefCookie);
     }
-}
\ No newline at end of file
+}
```

Comment:
I though all the line endings would have been normalised as part of #2504?

Reply:
Yeah, it should have.
For some stupid reason Visual Studio doesn't apply the `insert_final_newline` rule unless you open all files and save them manually.
I'll write a quick program to fix this.

Reply:
Only if you really want to. Fine to leave as is.

Reply:
So you're fine with this PR getting merged?

Reply:
> Only if you really want to. Fine to leave as is.

Rather not have these sorts of changes show up in unrelated commits/PR in the future